src/ucp/rndv/proto_rndv.c

```diff
@@ -396,6 +390,21 @@ static size_t ucp_proto_rndv_thresh(const ucp_proto_init_params_t *init_params)
     return cfg->rndv_thresh;
 }
 
+ucs_status_t
+ucp_proto_rndv_ctrl_am_init(ucp_proto_rndv_ctrl_init_params_t *params)
+{
+    /* Find lane to send the initial message */
+    params->lane = ucp_proto_common_find_am_bcopy_hdr_lane(
+                              &params->super.super);
+    if (params->lane == UCP_NULL_LANE) {
+        return UCS_ERR_NO_ELEM;
+    }
+
+    ucs_assert(params->super.send_op == UCT_EP_OP_AM_BCOPY);
+
+    return ucp_proto_rndv_ctrl_init(params);
```

Comment:
it is consistent with other params. 
also would need to add the same code (finding am lane and checking for errors) to many places

Reply:
we still keep ucp_proto_rndv_ctrl_am_init function to avoid code duplication, like this
```c
ucp_proto_rndv_ctrl_am_init(const ucp_proto_rndv_ctrl_init_params_t *params)
{
     /* Find lane to send the initial message */
    ucp_lane_index_t lane;

    lane  = ucp_proto_common_find_am_bcopy_hdr_lane(
                               &params->super.super);
     if (lane == UCP_NULL_LANE) {
         return UCS_ERR_NO_ELEM;
     }

     ucs_assert(params->super.send_op == UCT_EP_OP_AM_BCOPY);

     return ucp_proto_rndv_ctrl_init(params, lane);
```